pandas/tests/arrays/string_/test_string.py

```diff
@@ -332,3 +338,26 @@ def test_memory_usage():
     series = pd.Series(["a", "b", "c"], dtype="string")
 
     assert 0 < series.nbytes <= series.memory_usage() < series.memory_usage(deep=True)
+
+
+def test_string_dtype_subclassing():
+    @register_extension_dtype
+    class MyExtensionDtype(StringDtype):
```

Comment:
Not a big deal, but could also use pd.StringDtype here and below and then not have to import it above